topics/single-cell/tutorials/scrna-data-formats/slides.html

```diff
@@ -0,0 +1,578 @@
+
+**Supported languages:** R
+
+**Supported packages:** seurat
+
+---
+
+### Single Cell Experiment (SCE)
+
+.pull-left[
+
+- R-based format provided by the Bioconductor package SingleCellExperiment
+
+- Integrates well with the Bioconductor ecosystem
+
+<br/><br/>
+
+- **File extensions:**
+  - .rds
+  - .rda
+  - .rdata
+
+]
+
+
+.pull-right[
+
+![Basic structure of single-cell data](../../images/scrna-data-formats/gene-expression-diagram.png)
+
+]
+
+???
+
+Single Cell Experiment (SCE) is another R-based format that is widely used within the Bioconductor ecosystem of tools for both processing and analysis of single-cell data.
+
+**Supported languages:** R
+
+**Supported packages:** SingleCellExperiment
+
+---
+
+### CellDataSet (CDS)
+
+.pull-left[
+
+- R-based format used by the Monocle package
+
+- Integrates well with the Monocle package for various types of analysis
+
+<br/><br/>
+
+- **File extensions:**
+  - .rds
+  - .rda
+  - .rdata
+
+]
+
+.pull-right[
+
+![Basic structure of single-cell data](../../images/scrna-data-formats/gene-expression-diagram.png)
+
+]
+
+???
+
+CellDataSet (CDS) is the last of the R-based formats that is commonly used within the Monocle package which contains tools for performing various types of analysis.
+
+**Supported languages:** R
+
+**Supported packages:** monocle
+
+---
+
+### Compressed Files
+
+.pull-left[
+
+- Some files may be compressed/archived
+
+- Requires decompression/extraction to process the data
+
+- May receive errors if left compressed
+
+<br/><br/>
+
+- **File extensions:**
+  - .zip
+  - .gz
+  - .tar
+  - .tar.gz
+
+]
+
+.pull-right[
+
+![Basic structure of single-cell data](../../images/scrna-data-formats/unzip-tool.png)
+
+]
+
+???
+
+It is common for single-cell data to be stored in a compressed format in order to reduce filesizes and make transferring the data simpler. It is important to recognise when a file is compressed as using compressed data in downstream tools may cause errors to occur.
+
+There are two forms of compression that are common:
+
+- Individual file compression
+- Archive of multiple files
+
+Individual files are typically compressed with zip (.zip) or gzip (.gz) and should be unzipped prior to processing.
+
+When data is stored in multiple files/folders they may be archived into a single file (commonly called a tarball), this is done with the TAR tool (.tar). Like individual file compression, you will need to untar/extract the .tar file before further processing.
+
+Finally, it is also common for multiple files/folders to both be compressed and archived into a single file, this is indicated with the extension .tar.gz. In this case the data will need to be **both** unzipped and then extracted before the data is available in it's original form.
+
+---
+
+# Sources of single-cell data
+
+.pull-center[
+
+*Where are some sources to acquire public single-cell data?*
+
+]
+
+---
+
+### Data sources
+
+Repository | Link to resource

```

Comment:
Are these general resources or some specific experiments? Some entries in the table contain links to data repos and some to specific studies. The first one links to an article. Please check the links again.

Reply:
They should be for general single-cell resources, I'll have a check over and revise the links!